features/attached_enable.feature

```diff
@@ -634,7 +637,7 @@ Feature: Enable command behaviour when attached to an Ubuntu Pro subscription
         """
         One moment, checking your subscription first
         Installing snapd
-        Updating package lists
+        Updating Livepatch package lists
```

Comment:
Looking at the snap module, should this be `Updating main package lists` ?

Reply:
Let me have a look and fix it.

Reply:
Solved.